### PR TITLE
[PageControl] Update swift example

### PR DESCRIPTION
**Pull request by codeman7**
Fix colors to be consistent and make page numbers more Swift.

I accidentally pushed edits from another branch on my previous PR that's why I opened a new one.

### CODE REVIEW

File: components/PageControl/examples/PageControlTypicalUseExample.swift
Comment (randallli) [inline components/PageControl/examples/PageControlTypicalUseExample.swift]: where did we get these colors?

Comment (codeman7) [reply components/PageControl/examples/PageControlTypicalUseExample.swift]: They were the colors you guys were already using. Just those two instances the "f" was lower case and I figure you want all the colors to be either all lowercase letters or all uppercase.

**Comment by willlarche** [comment]
We are now using MDCPalettes for colors. Find the corresponding color in
that lightweight dependency.

**Comment by codeman7** [comment]
screen shot 2017-09-14 at 9 36 35 am
screen shot 2017-09-14 at 9 36 10 am
There seems to be an issue using the MDCPalette in Swift. I assume this is affecting code in other places as well.

**Comment by willlarche** [comment]
Maybe it’s just autocomplete. Will it compile if you type?

**Comment by codeman7** [comment]
It says no error if I try and compile too. I have cleaned the project and the build folder, still no luck. Is it just me?

**Comment by willlarche** [comment]
I think that's just the autocomplete then, no? Seems to be working if you
can compile.

**Comment by codeman7** [comment]
I can't compile.

**Comment by ianegordon** [comment]
Hi @codeman7 
This appears to work as expected on my devbox.  (Xcode 9 ga)
Did your latest commit solve your build issue or are you still having problems compiling?

**Review by ianegordon — APPROVED** [review]
Waiting on a +1 from Cody to merge.

**Comment by codeman7** [comment]
It was just because I had other examples in the same project that had MDCPalette so that's what caused the conflict. Everything is good now.

**Pull request merged**
